### ISSUE TITLE
Accessibility issues: Light Theme styles and more

### ISSUE THREAD

**Issue by prupipho** [Microsoft/WebTemplateStudio#1678]
### Light Theme
- boxes should be white not grey but white.
- + to add pages doesn´t seem ok in black and white.

### More
- Focus on input boxes (set box to orange)
   - doesn´t seem to work on the _EXTENSION_ for _SELECT PAGES_ on the _RIGHTSIDE BAR_.
   - doesn´t seem to work at all on the _CLIENT_. It´s orange for some, white for others and nothing happens for _SELECT PAGES_ on the _RIGHTSIDE BAR_.
- Top bar doesn´t work with ENTER but with SPACE only (not related to Light Theme but accessibility).
- _RIGHTSIDEBAR >  SELECT PAGES > CLOSE ICON_ should read "Remove" not the current "Cancel" .
- Not sure about white svgs for icons. Feel like they have to be dark.
- Step `Add Optional Cloud Services` in `Create Web App`. When using the keyboard to use the app, `Learn more` link doesn´t seem to be accessible. _ENTER_ and _SPACE_ only open _App Service_ or _Cosmos DB_ modals. No matter if the focus is on the box or on the _Learn More_ link.

### Questions
- how to reorder Pages with keyboard? Currently the arrows to do so, are not focusable at all

![image](https://user-images.githubusercontent.com/1333036/104685046-83530180-56fa-11eb-8df9-923a85fc005a.png)

 ![GetImage (2)](https://user-images.githubusercontent.com/1333036/99356610-bb7cc600-28aa-11eb-99ec-2f0c63faf1ac.png)
